src/domain/chipConfigurationTypeDef.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Configuration of a Chip.
+ * @typedef {Object} ChipConfiguration
+ * @property {string} id The id of this chip
+ * @property {string} title The title of this chip
+ * @property {string} href The url pointing to the content
+ * @property {boolean} permanent Chip should be always visible
+ * @property {string} target One of `["modal", "external"]`
```

Comment:
alternative? `@property {'modal'| 'external'} target Whether the content is showed in the modal component or a new browser tab

Reply:
Good idea!